scripts/travisTest.sh

```diff
@@ -39,17 +39,3 @@ echo -e "\n"
 echo -e  "\033[1;34mOpen Liberty release:\033[0m \033[1;36m$release\033[0m"
 echo -e "\033[1;34mOpen Liberty build number:\033[0m \033[1;36m$number\033[0m" 
 echo -e "\033[1;34mJava version:\033[0m\033[1;36m$jv\033[0m"
-
```

Comment:
Why are these steps being removed from the `travisTest.sh` script? Just leaving these action in the `.travis.yml` file is not going to work for daily build runs.